lib/tasks/hyacinth/ci.rake

```diff
@@ -1,7 +1,13 @@
+FEDORA_JETTY_ZIP_BASENAME = 'hyacinth-fedora-3.8.1-no-solr'.freeze
```

Comment:
This isn't all that important for the PR, but I'm thinking that this constant should probably just be a full URL.  I don't think there's any real benefit to storing the basename separately here.